### PR TITLE
Don't test field if both expected & actual are None

### PR DESCRIPTION
**Pull request by KhoomeiK**
We don't want to count these cases towards either the success rate or the failure rate.

### CODE REVIEW

**Review by asim-shrestha — APPROVED** [review]

File: tests/test_evals.py [hunk 1]
```diff
@@ -101,3 +101,17 @@ def test_validate_field_match_pass(expected, actual, field):
...
+    [
+        ({"field": None}, {"field": None}, "field"),
+        ({"field": None}, {"field": ""}, "field"),
+    ],
+)
+def test_validate_field_match_skip(expected, actual, field):
+    with pytest.raises(pytest.skip.Exception) as exc_info:
+        validate_field_match(expected, actual, field)
+
+    assert "Skipping this test" in str(exc_info.value)
```
Comment (asim-shrestha) [inline tests/test_evals.py:117]: Should add tests for when the field is None and the response is not None
